stock_inventory/models/stock_inventory.py

```diff
@@ -191,24 +191,56 @@ def _get_domain_category_quants(self, base_domain):
             ]
         )
 
+    def _get_products(self):
+        self.ensure_one()
+        if self.product_selection == "all":
+            return self.env["product.product"].search([])
+        elif self.product_selection == "category" and self.category_id:
+            return self.env["product.product"].search(
+                [("categ_id", "=", self.category_id.id)]
+            )
+        else:
+            return self.product_ids
+
+    def _get_overlapping_inventories(self):
+        for rec in self:
+            inventories = self.search(
+                [("state", "=", "in_progress"), ("id", "!=", rec.id)]
+            )
```

Comment:
```suggestion
            inventories = self.search(
                [("state", "=", "in_progress"), ("id", "!=", rec.id)]
            )
            if not inventories:
                continue
```

just a small improvement, sorry for the nitpicking 😆 for the rest LGTM!

Reply:
the more nitpicking the more quality :)